.drone.env

```diff
@@ -1,3 +1,3 @@
 # The test runner source for UI tests
-WEB_COMMITID=5367936f30b07c0832fb8af09f9f7719cccf7038
-WEB_BRANCH=master
+WEB_COMMITID=9b9616b8429432002d06c4a2c8f252cbe08fb735
+WEB_BRANCH=changeResponseCodeDelete
```

Comment:
Should this really be part of the PR?

Reply:
Ok. This seems to be needed to resolve a circular dependency with web. Needs to be revert once the related web changes are merged.